src/pacdef/aur_helper.py

```diff
@@ -67,8 +67,11 @@ def remove(self, packages: list[Package]) -> None:
 
         :param packages: list of packages to be removed.
         """
+        config = Config()
+        aur_helper_rm_args: list[str] = config.aur_helper_rm_args or []
+        switches_remove: list[str] = list(set(Switches.remove + aur_helper_rm_args))
```

Comment:
You go through a `set`, apparently to ensure switches only appear once. But that gets rid of the order. Pacman-wrapping AUR helpers need to specify the mode of operation (`-R` / `--remove`), (`-S` / `--sync`) first. So this would lead to invalid arguments. I would recommend to leave out the set.